practices/api_first_development

```diff
@@ -0,0 +1,94 @@
+---
+slug: api_first_development
+stages:
+  - development
+short_description: API-first approach to building products is rising, API first enables developers to innovate in ways that weren't before feasible and speed up the achievement of corporate goals.
+tags:
+  - Rest API
+  - Restful API
+  - API team
+  - API development
+  - API design
+  - Http
+keywords:
+  - restful api
+  - http methods
+  - api endpoints
+  - json
+  - api design
+  - api documentation
+author_github_username: D-stevpm
+---
+
+# API-First Development
+
+**TL;DR**
+
+API First is not a destination but a journey, You are API first when you prioritize APIs and add the daily incremental effort to make sure you strengthen your commitment to doing APIs. Among the different types of web APIs, REST (Representational State Transfer) API has become the most popular due to its simplicity, scalability, and flexibility.
+
+## What is API First Development 
+
+***Kin Lane Postman Chief Evangelist is the author of these words*** >API first design is API first, API first code is API first. It is important to note that your business is API First and you don’t actually need to tick a checkbox in other to attain API first-led company. 
+
+Let's take a step and explore what is API First - An API First approach posits that application programming interface (API) design and development should come before implementation.
+
+It is a methodology for developing software that gives teams justifications for putting the API (Application Programming Interface) at the center of the procedure. This means that before working on the user interface (UI) or other application components, developers concentrate on designing, creating, and testing the API.
+
+Remember earlier we mentioned that API First is not a destination but a journey and that there is no checklist you must follow in order to publicly refer to your company as an API First company, You'll undoubtedly notice that your software team spent a lot of time developing, testing, and ensuring that the API achieves business objectives when it comes time to adopt APIs across other products. This is because APIs come first in your company. I now advise you to consider your software development methodology and never doubt presenting your business as an API.
```

Comment:
```suggestion
Remember earlier we mentioned that API First is not a destination but a journey and that there is no checklist you must follow in order to publicly refer to your company as an API First company. You'll undoubtedly notice that your software team spent a lot of time developing, testing, and ensuring that the API achieves business objectives when it comes time to adopt APIs across other products. This is because APIs come first in your company. I now advise you to consider your software development methodology and never doubt presenting your business as an API.
```